install.sh

```diff
@@ -3,3 +3,6 @@
 set -e
 
 sudo apt-get install -y curl awscli
+
+ln -s ${GEOCINT_WORK_DIRECTORY}/completeness_report/ ${GEOCINT_WORK_DIRECTORY}/public_html/completeness_report/
```

Comment:
it's install.sh, and it installs dependancies.
is `${GEOCINT_WORK_DIRECTORY} /geocint` created already? or it is created when geocint pipeline is started first time?

Also, it seems that `${GEOCINT_WORK_DIRECTORY}/completeness_report/ `shoud be rather `${GEOCINT_WORK_DIRECTORY}/**geocint**/completeness_report/`

Reply:
no need to have folders to create symbolic link. tested on stratum